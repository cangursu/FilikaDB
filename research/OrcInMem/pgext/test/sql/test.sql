--select orc_buffer_test('/home/postgres/sock_orcinmem');
--select orc_inmem_test('/home/postgres/sock_orcinmem', '/home/postgres/projects/filikadb/research/OrcInMem/pgext/test/sql/data/');
select orc_buffer_remote_test('/home/postgres/sock_orcinmem',  '/home/postgres/projects/filikadb/research/OrcInMem/MemStreamServer/remotesock');