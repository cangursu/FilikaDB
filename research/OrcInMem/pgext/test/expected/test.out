select orc_buffer_test('/home/postgres/sock_orcinmem');
     orc_buffer_test     
-------------------------
 orc_buffer_test SUCCEED
(1 row)

select orc_inmem_test('/home/postgres/sock_orcinmem', '/home/postgres/projects/filikadb/research/OrcInMem/pgext/test/sql/data/');
     orc_inmem_test     
------------------------
 orc_inmem_test SUCCEED
(1 row)

